src/os/bluestore/BlueStore.cc

```diff
@@ -3325,6 +3325,8 @@ BlueStore::Onode* BlueStore::Onode::decode(
   auto p = v.front().begin_deep();
   on->onode.decode(p);
   for (auto& i : on->onode.attrs) {
+    //not sure whether this will go to onode_cache or cache_other; leaving
```

Comment:
attrs are part of the onode, so I think it would make sense to include them in cache_onode